data/en/structfind.json

```diff
@@ -1,13 +1,14 @@
 {
 	"name":"structFind",
 	"type":"function",
-	"syntax":"structFind(structure, key)",
+	"syntax":"structFind(structure, key [, defaultValue ])",
 	"returns":"any",
-	"related":[],
-	"description":" Determines the value associated with a key in a structure.",
+	"related":["structFindKey", "structFindValue"],
+	"description":"Determines the value associated with a key in a structure.",
 	"params": [
-		{"name":"structure","description":"Structure that contains the value to return","required":true,"default":"","type":"struct","values":[]},
-		{"name":"key","description":"Key whose value to return","required":true,"default":"","type":"string","values":[]}
+		{"name": "structure", "description": "Structure that contains the value to return", "required": true, "default": "", "type": "struct", "values": []},
+		{"name": "key", "description": "Key whose value to return", "required": true, "default": "", "type": "string", "values": []},
+		{"name": "defaultValue", "description": "Default value which will be returned if the key does not exist or if null was found. Currently only supported by Lucee. See http://docs.lucee.org/reference/functions/structfind.html#argument-defaultValue", "required": false, "default": "", "type": "any"}
 
 	],
 	"engines": {
```

Comment:
And yes, you were right: Mentioning this in the engine notes for Lucee is a good idea :+1: